### PR TITLE
Solution to 8- Hangman 08

### LINKED ISSUE #21: 8 - Hangman-08
Create a constant to hold an integer value representing the highest number of failed attempts a user can make. Set the value to any integer you want.

Using this constant, make the game track the number of failed attempts such that a message is shown and the game stops when the failed attempts exceed the value in the constant.

Extra:
Read about classes and objects in python.

### PR DESCRIPTION
**Pull request by ebosetalee**
Resolves #21 [https://repl.it/join/grcgbasj-emmanuellaesezo](https://repl.it/join/grcgbasj-emmanuellaesezo)
[https://repl.it/@EmmanuellaEsezo/8-Hangman-08](https://repl.it/@EmmanuellaEsezo/8-Hangman-08)

### CODE REVIEW

File: hangman/hangman.py [hunk 1]
```diff
@@ -21,9 +21,11 @@ def hangman():
     Processes the user input, with the target word
     to determine which letter is guessed correctly.
     """
+    times = 10
```
Comment (yudori) [inline hangman/hangman.py]: Use a descriptive name.

File: hangman/hangman.py [hunk 2]
```diff
@@ -21,9 +21,11 @@ def hangman():
...
     word=""
     count = 0
     while True:
+        print(count)
```
Comment (yudori) [inline hangman/hangman.py]: Either remove or print actual information that is useful to the user

File: hangman/hangman.py [hunk 3]
```diff
@@ -36,6 +38,11 @@ def hangman():
             for val in dic_list:
                 print(val + ": {} ".format(dic_list[val]))
             break
+        elif count == times:
+            print("The END")
```
Comment (yudori) [inline hangman/hangman.py]: Na wa oo, carry your users along na. Why will I be playing game and suddenly see "The end" with no reason why the game is over?

**Review by yudori — CHANGES_REQUESTED** [review]

File: hangman/hangman.py (hunk 1 above)
Comment (ebosetalee) [reply hangman/hangman.py]: ok

File: hangman/hangman.py (hunk 2 above)
Comment (ebosetalee) [reply hangman/hangman.py]: This was a mistake,
 Used it for testing.

File: hangman/hangman.py (hunk 3 above)
Comment (ebosetalee) [reply hangman/hangman.py]: Lol, sorry.

File: hangman/hangman.py [hunk 4]
```diff
@@ -21,11 +21,11 @@ def hangman():
     Processes the user input, with the target word
     to determine which letter is guessed correctly.
     """
-    times = 10
+    attempt = 10
     word=""
     count = 0
     while True:
-        print("You have {} attempts".format(times-count))
+        print("You have {} attempts".format(attempt-count))
```
Comment (yudori) [inline hangman/hangman.py]: Did you even test this?

File: hangman/hangman.py [hunk 5]
```diff
@@ -38,7 +38,7 @@ def hangman():
             for val in dic_list:
                 print(val + ": {} ".format(dic_list[val]))
             break
-        elif count == times:
+        elif count == attempt:
```
Comment (yudori) [inline hangman/hangman.py]: I specifically said to track FAILED ATTEMPTS.

Comment (ebosetalee) [reply hangman/hangman.py]: I don't understand
Is failed as in the wrong input?

Comment (yudori) [reply hangman/hangman.py]: When they make a wrong guess, it's a failed attempt.

**Review by yudori — APPROVED** [review]

**Review by jirevwe — APPROVED** [review]

**Pull request merged**